client/src/components/Chat/MessageCreator.js

```diff
@@ -92,6 +114,9 @@ const MessageCreator = () => {
                     className={classes.sendButton} 
                     variant='outlined' 
                     color='secondary'
+                    onClick={() => {
```

Comment:
Would be nice if the input had a "onKeyUp" event to send the message for when the Enter key is pressed

Reply:
Also if you scrolled down the window when each message is sent or received

Reply:
Added scroll to bottom on  new message, and "onKeyUp" event to send messages with the Enter key.